package.json

```diff
@@ -22,6 +22,7 @@
     "@ng-bootstrap/ng-bootstrap": "^13.0.0",
     "@popperjs/core": "^2.11.6",
     "bootstrap": "^5.2.0",
+    "bootstrap-icons": "^1.11.3",
```

Comment:
I am okay adding additional packages to the project, but we should all talk it over before we commit to one. Where is this used in the project? Is that in the style file for your new component?